src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/events/cas2/Cas2StatusFactory.kt

```diff
@@ -3,11 +3,17 @@ package uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.events.cas2
 import io.github.bluegroundltd.kfactory.Factory
 import io.github.bluegroundltd.kfactory.Yielded
 import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.cas2.model.Cas2Status
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.api.events.cas2.model.StatusDetailItem
 
 class Cas2StatusFactory : Factory<Cas2Status> {
   private var name: Yielded<String> = { "moreInfoRequested" }
   private var label: Yielded<String> = { "More information requested" }
   private var description: Yielded<String> = { "More information about the application has been requested" }
+  private var statusDetails = listOf(
+    StatusDetailItem("exclusionZonesAndAreas", "Exclusion zones and preferred areas"),
+    StatusDetailItem("riskOfSeriousHarm", "Risk of serious harm"),
```

Comment:
quite minor but I wonder as this field is potentially empty, if for the factory default we set it as an `emptylist()`, and then users of the factory can have a `withStatusDetails` function to add them if needed, otherwise we might be populating unecessary/unnoticed data

Reply:
Changed as per your comment in a more recent commit.